org.lflang/src/org/lflang/generator/c/CCmakeCompiler.java

```diff
@@ -45,6 +44,10 @@
  * @author Soroush Bateni <soroush@utdallas.edu>
  */
 class CCmakeCompiler extends CCompiler {
+    // FIXME: This does not override the compileCCommand
+    //  method because there is no single command that
+    //  compiles; however, it is not clear why
+    //  compileCCommand is public in the first place.
```

Comment:
Because it is also used in `org.lflang.federated.FedCLauncher`.

Reply:
Oops, noted. I will look at that. In hindsight it was probably wrong to sneak these changes into this branch. I'll be more careful next time.